styles.css

```diff
@@ -150,3 +177,19 @@ input {
 .past-activities {
   text-align: center;
 }
+
+@media only screen and (max-width: 1300px) {
```

Comment:
These are interesting to see since we have not even learned about these. So advanced!